### PR TITLE
Set up linting for repo (typechecking + eslint)

### PR DESCRIPTION
**Pull request by KaltunAhmed**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,19 +5,26 @@
     "dev": "next dev -p 4000",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "typecheck": "tsc",
+    "lint:eslint": "eslint --ext .ts,.tsx"
```
Comment (krivachy) [inline package.json:10]: Do we need both `next lint` + `eslint`?

**Comment by krivachy** [comment]
Also: will you be adding a github action as a separate PR?

File: package.json (hunk 1 above)
Comment (KaltunAhmed) [reply package.json:10]: I guess we can remove next lint

**Comment by KaltunAhmed** [comment]
> Also: will you be adding a github action as a separate PR?

Ahhhh yes of course. No let me do it in this PR

**Review by mattvagni — APPROVED** [review]

**Pull request merged**